mrjonhazlett-Omaha_NE.md

```diff
@@ -13,3 +13,4 @@
 - TD Ameritrade Park (:baseball)
 - Old Market
 - Pedestrian Bridge
```

Comment:
Omaha is about the size of Riverside. Lots of indoor things to do (mostly bars)... but they also have the best zoo in the world (outranks San Diego Zoo), baseball stadiums, and Husker football!